src/api-request/index.ts

```diff
@@ -1,8 +1,8 @@
 import superagent from 'superagent';
```

Comment:
I would leave the api as a single file and not to create its own folder until it's worth it.